<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>history_Dialog</class>
 <widget class="QDialog" name="history_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>History Record</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clearPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletePushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="shortcut">
        <string>Del</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="usePushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Use</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="overwriteMode">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>usePushButton</tabstop>
  <tabstop>clearPushButton</tabstop>
  <tabstop>copyPushButton</tabstop>
  <tabstop>deletePushButton</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>settext()</slot>
 </slots>
</ui>
